techlibs/quicklogic/Makefile.inc

```diff
@@ -1,4 +1,4 @@
-techlibs/quicklogic/qlf_k6n10f/bram_types_sim.v: techlibs/quicklogic/qlf_k6n10f/generate_bram_types_sim.py
```

Comment:
I am not sure we want this. Like all generated files this should be generated in the out-of-tree build directory so that the source tree stays clean. What problem is this fixing?

Reply:
That's what I thought too, but then I saw this:
techlibs/quicklogic/qlf_k6n10f/.gitignore:1:/bram_types_sim.v
So I guess  bram_types_sim.v is supposed to be generated in techlibs/quicklogic/qlf_k6n10f/

Reply:
Pretty sure this broke the Wasm build before, even.

Reply:
> So I guess bram_types_sim.v is supposed to be generated in techlibs/quicklogic/qlf_k6n10f/

I wouldn't take that to imply `bram_types_sim.v` must be generated there, it's just that it can be unless you are doing an out-of-tree build. In the case of out-of-tree I would expect it to be placed at the same path but relative to the build directory. If there's an issue I think we should fix the rule that takes `bram_types_sim.v` for its input, not this rule that produces it.

Reply:
Done, now generated files get generated in the build dir

Reply:
Great!